build.gradle.kts

```diff
@@ -139,13 +171,14 @@ subprojects {
 }
 
 apply {
-
-    // Aggregated coverage report across all subprojects.
-    from(Scripts.jacoco(project))
-
     // Generate a repository-wide report of 3rd-party dependencies and their licenses.
     from(Scripts.repoLicenseReport(project))
 
     // Generate a `pom.xml` file containing first-level dependency of all projects in the repository.
     from(Scripts.generatePom(project))
 }
+
+afterEvaluate {
```

Comment:
Why `afterEvaluate` is required? Are we doing some black magic here? At least it may be worth documenting this part.

Reply:
Added an explanation with a link to a Gradle discussion.